accord-core/src/test/java/accord/impl/basic/Cluster.java

```diff
@@ -143,23 +142,30 @@ public static void run(Id[] nodes, Supplier<PendingQueue> queueSupplier, Consume
     {
         Shards shards = topologyFactory.toShards(nodes);
         Map<Id, Node> lookup = new HashMap<>();
-        Cluster sinks = new Cluster(queueSupplier, lookup::get, responseSink, stderr);
-        for (Id node : nodes)
-            lookup.put(node, new Node(node, shards, shards.forNode(node), sinks.create(node, randomSupplier.get()),
-                                      randomSupplier.get(), nowSupplier.get(), ListStore::new, ListAgent.INSTANCE, sinks));
-
-        List<Id> nodesList = new ArrayList<>(Arrays.asList(nodes));
-        sinks.recurring(() ->
+        try
         {
-            Collections.shuffle(nodesList, randomSupplier.get());
-            int partitionSize = randomSupplier.get().nextInt((topologyFactory.rf+1)/2);
-            sinks.partitionSet = new HashSet<>(nodesList.subList(0, partitionSize));
-        }, 5L, TimeUnit.SECONDS);
-
-        Packet next;
-        while ((next = in.get()) != null)
-            sinks.add(next);
-
-        while (sinks.processPending());
+            Cluster sinks = new Cluster(queueSupplier, lookup::get, responseSink, stderr);
+            for (Id node : nodes)
+                lookup.put(node, new Node(node, shards, sinks.create(node, randomSupplier.get()),
```

Comment:
We need to create something besides a SingleThread CommandStore here, for determinism.